simulator-gradle-plugin/build.gradle.kts

```
@@ -32,3 +32,8 @@ dependencies {
   compileOnly(libs.android.gradle.api)
   implementation(libs.kotlin.stdlib)
 }
+
+tasks.withType<org.gradle.plugin.devel.tasks.ValidatePlugins> {
+  failOnWarning.set(true)
+  enableStricterValidation.set(true)
+}
```

You can make it friendlier by using the Kotlin extensions provided by Gradle:
```suggestion
tasks.validatePlugins {
  failOnWarning = true
  enableStricterValidation = true
}
```